<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>481</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::West</enum>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <widget class="QWidget" name="tab_main">
     <attribute name="icon">
      <iconset>
       <normaloff>../build-kwix_project-Desktop-Debug/img/home.png</normaloff>../build-kwix_project-Desktop-Debug/img/home.png</iconset>
     </attribute>
     <attribute name="title">
      <string/>
     </attribute>
     <widget class="QLabel" name="label_home">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>701</width>
        <height>361</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../build-kwix_project-Desktop-Debug/main_screen.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_QR">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>390</y>
        <width>171</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>360</y>
        <width>191</width>
        <height>111</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'PibotoLt'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Touch and Scan QR Code&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_st_in">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>360</y>
        <width>99</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string>NEW</string>
      </property>
     </widget>
     <zorder>label_home</zorder>
     <zorder>textEdit</zorder>
     <zorder>lineEdit_QR</zorder>
     <zorder>btn_st_in</zorder>
    </widget>
    <widget class="QWidget" name="tab_elisa">
     <attribute name="icon">
      <iconset>
       <normaloff>../build-kwix_project-Desktop-Debug/img/operation.png</normaloff>../build-kwix_project-Desktop-Debug/img/operation.png</iconset>
     </attribute>
     <attribute name="title">
      <string/>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_back">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>50</y>
        <width>231</width>
        <height>291</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="VideoLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>651</width>
        <height>291</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>124</red>
            <green>124</green>
            <blue>124</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>124</red>
            <green>124</green>
            <blue>124</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>124</red>
            <green>124</green>
            <blue>124</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="AlternateBase">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>124</red>
            <green>124</green>
            <blue>124</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>124</red>
            <green>124</green>
            <blue>124</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>124</red>
            <green>124</green>
            <blue>124</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="AlternateBase">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>124</red>
            <green>124</green>
            <blue>124</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>124</red>
            <green>124</green>
            <blue>124</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>124</red>
            <green>124</green>
            <blue>124</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="AlternateBase">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(0, 0, 0);
background-color: rgb(124, 124, 124);
font: 75 23pt &quot;나눔고딕&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>90</y>
        <width>191</width>
        <height>201</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLineEdit" name="lineEdit_HUE">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Hue : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_Sat">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Sat : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_RGB">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>RGB : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_Lum">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Lum: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_Result">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Result :</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="btn_measure">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>290</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>MEASURE</string>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>350</y>
        <width>531</width>
        <height>101</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="btn_start">
         <property name="text">
          <string>START</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="btn_backward">
         <property name="text">
          <string>BACKWARD</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="btn_forward">
         <property name="text">
          <string>FORWARD</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btn_pause">
         <property name="text">
          <string>STOP</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="btn_save">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>360</y>
        <width>151</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>SAVE</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_FPS">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>60</y>
        <width>151</width>
        <height>30</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>FPS : </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Sequence">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>300</y>
        <width>401</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_Timer">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>191</width>
        <height>30</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(139, 139, 139);</string>
      </property>
      <property name="text">
       <string>Timer:  15 minutes</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_userinfo">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>681</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>USER : </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_covid">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>50</y>
        <width>189</width>
        <height>32</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_covid">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>50</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_state">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>280</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>19</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <zorder>VideoLabel</zorder>
     <zorder>lineEdit_back</zorder>
     <zorder>verticalLayoutWidget</zorder>
     <zorder>btn_measure</zorder>
     <zorder>gridLayoutWidget_2</zorder>
     <zorder>btn_save</zorder>
     <zorder>label_FPS</zorder>
     <zorder>label_Timer</zorder>
     <zorder>label_userinfo</zorder>
     <zorder>label_Sequence</zorder>
     <zorder>lineEdit_covid</zorder>
     <zorder>label_covid</zorder>
     <zorder>label_state</zorder>
    </widget>
    <widget class="QWidget" name="tab_setting">
     <attribute name="icon">
      <iconset>
       <normaloff>../build-kwix_project-Desktop-Debug/img/settings.png</normaloff>../build-kwix_project-Desktop-Debug/img/settings.png</iconset>
     </attribute>
     <attribute name="title">
      <string/>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>681</width>
        <height>351</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="1">
        <widget class="QSpinBox" name="spinBox_Reaction">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>180</number>
         </property>
         <property name="value">
          <number>40</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_enzyme_time">
         <property name="text">
          <string>REACTION Time</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_antibody_time">
         <property name="text">
          <string>TMB Time</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_min_1">
         <property name="text">
          <string>min</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="spinBox_Tmb">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>180</number>
         </property>
         <property name="value">
          <number>15</number>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="label_min_2">
         <property name="text">
          <string>min</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_location">
         <property name="text">
          <string>Location</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="location_box">
         <item>
          <property name="text">
           <string>Seoul</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>jeju-do</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Gangwon-do</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Incheon</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Busan</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sejong</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Daegu</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Gwangju</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Daejeon</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ulsan</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Gyeonggi-do</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Chungcheongnam-do</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Chungcheongbuk-do</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Jeollanam-do</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Jeollabuk-do</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Gyeongsangnam-do</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Gyeongsangbuk-do</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>370</y>
        <width>561</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="btn_set_save">
         <property name="text">
          <string>SAVE</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_result">
     <attribute name="icon">
      <iconset>
       <normaloff>../build-kwix_project-Desktop-Debug/img/results.png</normaloff>../build-kwix_project-Desktop-Debug/img/results.png</iconset>
     </attribute>
     <attribute name="title">
      <string/>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>681</width>
        <height>451</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableView" name="tableView"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>WEB UPLOAD</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_refresh">
         <property name="text">
          <string>REFRESH</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
